js/page-specific/swiftype-search-results.js

```diff
@@ -8,13 +8,21 @@ baltimoreCounty.pageSpecific.swiftypeSearchResults = (function swiftypeSearchRes
 
 	function getSearchResults(searchTerm, pageNumber) {
 		var currentPageNumber = pageNumber || 1;
-		var cleanedSearchTerm = searchTerm.replace(/\+/g, '%20');
+		var cleanedSearchTerm = searchTerm;
 
-		$.ajax(constants.keywordSearch.urls.api + cleanedSearchTerm + '/' + currentPageNumber).then(searchResultRequestSuccessHandler, searchResultRequestErrorHandler);
+		// Ensure the last character is not '+' as the the trailing space causes no results
+		if (cleanedSearchTerm[cleanedSearchTerm.length - 1] === '+') {
+			cleanedSearchTerm = cleanedSearchTerm.substr(0, (cleanedSearchTerm.length - 1));
```

Comment:
Unnecessary, but you could also do **cleanedSearchTerm.slice(0, -1)** which is a little more readable.